src/vsr/superblock.zig

```diff
@@ -1401,37 +1401,6 @@ pub fn SuperBlockType(comptime Storage: type) type {
             assert(offset + size <= superblock_zone_size);
         }
 
-        /// We use flexible quorums for even quorums with write quorum > read quorum, for example:
-        /// * When writing, we must verify that at least 3/4 copies were written.
-        /// * At startup, we must verify that at least 2/4 copies were read.
-        ///
-        /// This ensures that our read and write quorums will intersect.
-        /// Using flexible quorums in this way increases resiliency of the superblock.
-        fn threshold_for_caller(caller: Caller) u8 {
```

Comment:
This sounds like an important code that perhaps _should_ be used?

Reply:
This code lives on in `src/vsr/superblock_quorums.zig`!